### ISSUE TITLE
RPC::DEADLINE_EXCEEDED: context deadline exceeded

### ISSUE THREAD

**Issue by Saitejacity** [GoogleChrome/lighthouse#16288]
Hi everyone,

I tried to test the speed of my site https://www.kugans.com/ using https://pagespeed.web.dev/ but every time I am getting a error

 RPC::DEADLINE_EXCEEDED: context deadline exceeded.

I tried on pingdom, GTmatrix and it is working well and showing the results.

Can anyone help how this can be fixed.

Thank you for your help?

**Comment by adamraine** [comment]
Yeah I can repro: https://pagespeed.web.dev/analysis/https-www-kugans-com/67qb2zmhzg?form_factor=mobile

**Comment by Saitejacity** [comment]
Hi 

Can anyone help me on this please?

Thank you

**Comment by boy70263** [comment]
Hi I have same question , my website: https://www.chinatimes.com/newspapers/20210623000177-260203

somettimes work well but sometimes not work

not work：
https://pagespeed.web.dev/analysis/https-www-chinatimes-com-newspapers-20210623000177-260203/p2jru6um4r?form_factor=mobile

https://pagespeed.web.dev/analysis/https-www-chinatimes-com-newspapers-20210623000177-260203/1ndby56e6z?form_factor=mobile

use apikey
"code": 500,
"message": "Lighthouse returned error: Something went wrong.",
 
"code": 500,
"message": "Lighthouse returned error: PAGE_HUNG. Lighthouse was unable to reliably load the URL you requested because the page stopped responding.",


work well：

https://pagespeed.web.dev/analysis/https-www-chinatimes-com-newspapers-20210623000177-260203/xgbvdkwwss?form_factor=mobile

use apikey
[runPagespeed.json](https://github.com/user-attachments/files/18224428/runPagespeed.json)

**Comment by Saitejacity** [comment]
> Hi
> 
> Can anyone help me on this please? closed by mistake.
> 
> Thank you

**Comment by Saitejacity** [comment]
Is there any update on this please ?

**Comment by 12Aleks** [comment]
Hi, I have a similar problem, with my portfolio site [https://www.frontendpolska.pl/pl](https://www.frontendpolska.pl/pl) , I can't figure out what the problem is with Lighthouse it works fine. The site is indexed. I used the next 14.2.15 and the next-intel 3.22 maybe the problem is related to redirects in language versions.

**Comment by MariuszMilka** [comment]
Same problem here. RPC::DEADLINE_EXCEEDED: context deadline exceeded on a new site. Same site was in development on the same server and the old copy worked. Only once I moved to a new domain on the same server, it stopped Lighthouse stopped working.

**Comment by umair68** [comment]
Same issue here.

**Comment by sankalpsans** [comment]
+1 same issue on multiple websites. e.g. https://pagespeed.web.dev/analysis/https-www-moneycontrol-com-news-business-apple-partner-hon-hai-injects-1-5-billion-into-india-unit-13033249-html/xa4ae3lc5p?form_factor=mobile

**Comment by felisucoibi** [comment]
I started to have same problem, maybe is because i'm using href and the api of pagespeed? ¡Vaya! Se ha producido un error.
RPC::DEADLINE_EXCEEDED: context deadline exceeded
https://justdust.com

UPDATE:
Not sure what i did to fix it but i did. I reviewed my WAF rules that blocked robots.txt also i uninstalled and installed again all my wp plugins just in case somebody find this information useful.

**Comment by qalqi** [comment]
+1
